spec/lib/convenient_service/core/class_methods_spec.rb

```diff
@@ -141,7 +141,7 @@ def foo(*args, **kwargs, &block)
           ##
           # NOTE: Intentionally calling missed method that won't be included (no concerns with it).
           #
-          expect { service_class.foo }.to raise_error(NoMethodError).with_message("undefined method `foo' for #{service_class}")
+          expect { service_class.foo }.to raise_error(NoMethodError).with_message(/undefined method `foo' for #{service_class}/)
```

Comment:
**FYI**

After the reraising of the original error, `DidYouMean` started to add its suggestions, which is why the string is replaced by regexp.